### PR TITLE
Refresh excluding packages and files

### PR DESCRIPTION
**Pull request by rolandtritsch**
Hi @ckipp01,

here is the second PR on refreshing the excluded packages and files functionality.

### CODE REVIEW

File: src/sbt-test/scoverage/coverage-excluded-files/build.sbt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+libraryDependencies += "org.scalameta" %% "munit" % "0.7.29" % Test
+
+coverageExcludedFiles := ".*\\/two\\/GoodCoverage\\.scala"
+ 
+resolvers ++= {
+  if (sys.props.get("plugin.version").exists(_.endsWith("-SNAPSHOT")))
+    Seq(Resolver.sonatypeRepo("snapshots"))
+  else Seq.empty
+}
```
Comment (rolandtritsch) [inline src/sbt-test/scoverage/coverage-excluded-files/build.sbt:13]: Adding a test for `excludedFiles`.

File: src/sbt-test/scoverage/coverage-excluded-packages/build.sbt [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+libraryDependencies += "org.scalameta" %% "munit" % "0.7.29" % Test
+
+coverageExcludedPackages := "two\\..*"
+ 
+resolvers ++= {
+  if (sys.props.get("plugin.version").exists(_.endsWith("-SNAPSHOT")))
+    Seq(Resolver.sonatypeRepo("snapshots"))
+  else Seq.empty
+}
```
Comment (rolandtritsch) [inline src/sbt-test/scoverage/coverage-excluded-packages/build.sbt:13]: Adding test for `excludedPackages`.

File: src/sbt-test/scoverage/coverage-off/test [hunk 3]
```diff
@@ -7,4 +7,4 @@
 > coverageOff
 > test
 # There should be no scoverage-data directory
--$ exists target/scala-2.12/scoverage-data
+-$ exists target/scala-2.13/scoverage-data
```
Comment (rolandtritsch) [inline src/sbt-test/scoverage/coverage-off/test:10]: Fixing typo, Making test work (again).

File: src/sbt-test/scoverage/scala3-coverage-excluded-files/test [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# run scoverage using the coverage task
+> clean
+> coverage
+> test
+> coverageReport
+# There should be no directory for the excluded files
+#-$ exists target/scala-3.2.0-RC1/scoverage-report/two
+# But right now there is, because Scala3 does not support excluding files
+$ exists target/scala-3.2.0-RC1/scoverage-report/two
```
Comment (rolandtritsch) [inline src/sbt-test/scoverage/scala3-coverage-excluded-files/test:9]: Just to show/document that excluding files (and packages) is not working right now.

Aiming to make this work (somehow).

**Review by ckipp01 — APPROVED** [review]
Thanks a lot for the additional tests and documentation @rolandtritsch. LGTM

**Pull request merged**